datadog_checks_base/tests/base/utils/db/test_util.py

```diff
@@ -276,3 +277,19 @@ def test_dbm_async_job_inactive_stop(aggregator):
 def test_default_json_event_encoding(input):
     # assert that the default json event encoding can handle all defined types without raising TypeError
     assert json.dumps(input, default=default_json_event_encoding)
+
+
+@pytest.mark.parametrize("track_operation_time", [True, False])
+def test_tracked_query(aggregator, track_operation_time):
+    def test_query():
+        time.sleep(0.1)
```

Comment:
Can we mock the time in some way instead of adding pauses (e.g. by mocking `get_timestamp()`)? That would let you assert the actual value of the metric further below and the `sleep` here is not giving us anything here.